haskell/private/actions/package.bzl

```diff
@@ -64,7 +64,7 @@ def package(
       exposed_modules: List of exposed modules.
       other_modules: List of hidden modules.
       my_pkg_id: Package id object for this package.
-      has_hs_library: Whether hs-libraries should be non-null.
+      has_hs_library: Whether hs-libraries should be created.
```

Comment:
I thought about refactoring this function a bit further, but AFAICS it is part of the public interface of the haskell toolchain in `actions.package` I did not went further.